catalog/imaging/imaging-actionprovider-chip/src/main/java/org/codice/alliance/imaging/chip/actionprovider/ImagingChipActionProvider.java

```diff
@@ -0,0 +1,124 @@
+/**
+ * Copyright (c) Codice Foundation
+ * <p>
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ * <p>
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.alliance.imaging.chip.actionprovider;
+
+import java.net.MalformedURLException;
+import java.net.URL;
+import java.util.Arrays;
+import java.util.List;
+
+import org.codice.ddf.configuration.SystemBaseUrl;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.vividsolutions.jts.geom.Coordinate;
+import com.vividsolutions.jts.geom.Geometry;
+import com.vividsolutions.jts.io.ParseException;
+import com.vividsolutions.jts.io.WKTReader;
+
+import ddf.action.Action;
+import ddf.action.ActionProvider;
+import ddf.action.impl.ActionImpl;
+import ddf.catalog.data.Metacard;
+
+public class ImagingChipActionProvider implements ActionProvider {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(ImagingChipActionProvider.class);
+
+    public static final String TITLE = "Chip Image";
+
+    public static final String DESCRIPTION =
+            "Opens a new window to enter the boundaries of an image chip for a Metacard.";
+
+    public static final String PATH = "/chipping/chipping.html";
+
+    public static final String ID = "image-chipping-action-view";
+
+    public static final String NITF_CONTENT_TYPE = "image/nitf";
+
+    @Override
+    public <T> List<Action> getActions(T input) {
```

Comment:
I think this method is supposed to return an empty list if no actions can be performed.